app/src/main/java/app/grapheneos/camera/capturer/ImageSaver.kt

```diff
@@ -126,13 +126,14 @@ class ImageSaver(
     private fun saveImage() {
         try {
             saveImageInner()
+            imageCapturer.mActivity.thumbnailLoaderExecutor.executeIfAlive(this::generateThumbnail)
         } catch (e: ImageSaverException) {
             handleError(e)
-            return
+        } finally {
+            photosInQueue.getAndDecrement()
         }
         Log.i("TAG", "Done with image $photosInQueue")
-        photosInQueue.getAndDecrement()
-        imageCapturer.mActivity.thumbnailLoaderExecutor.executeIfAlive(this::generateThumbnail)
```

Comment:
This is not what I meant. 
`imageCapturer.mActivity.thumbnailLoaderExecutor.executeIfAlive(this::generateThumbnail)` should stay outside the `try {}` block, because it doesn't throw `ImageSaverException` (or any exception, really)

Reply:
Oh got it. Just pushed the changes. Thanks for the clarification!